docs/concepts/automations.md

```diff
@@ -145,9 +185,19 @@ triggers:
       - external.resource.pinged
     parameters:
       param_1: "{{ event }}"
+  - enabled: true
+    match:
+      prefect.resource.id: my.other.external.resource
+    expect:
+      - some.other.event
+    parameters:
+      param_1: "{{ event }}"
 ```
+Both of the above triggers would be attached to `test-deployment` after running `prefect deploy`.
+
 
-When applied, this will create a linked automation that responds to events from an external resource, and passes that event into the parameters of the executed flow run.
+!!! warning "Use either the `--trigger` flag ***or*** your `prefect.yaml` file to define triggers"
+  To avoid confusion or unexpected behavior, `prefect deploy` will raise an error if you attempt to use both the `--trigger` flag and a `prefect.yaml` to define triggers for a given deployment.
```

Comment:
I am confused by this. Why can't I just override any default triggers? This differs from the behavior of the deploy actions and I would find that distinction hard to remember 🤔  As a use case, if someone on my team puts a trigger in the `prefect.yaml` file, I can't use `prefect deploy --triggers` at all without deleting it and defining triggers every time imperatively, right?

Can you explain more? I might be missing context.

Reply:
overriding existing (in `prefect.yaml`) triggers makes sense to me, its where I started on this actually

im cool with going back to prioritizing triggers passed via CLI

Reply:
updated!